TrafficCapture/captureKafkaOffloader/src/test/resources/log4j2.properties

```diff
@@ -13,7 +13,7 @@ appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss,SSS}{UTC} %p %c{1.} [%t] %m%equals{ ctx=%mdc}{ ctx=\{\}}{}%n
 
 # Allow customization of owned package logs
-logger.rfs.name = com.rfs
+logger.rfs.name = org.opensearch.migrations.bulkload
```

Comment:
Odd to find this in here; any clue what's going on?

Reply:
Came from this PR [1]; separate issue

[1] https://github.com/opensearch-project/opensearch-migrations/pull/963